diff --git a/tools/toolchains/mbed_toolchain.py b/tools/toolchains/mbed_toolchain.py
index 51429c0012..74d6924874 100755
--- a/tools/toolchains/mbed_toolchain.py
+++ b/tools/toolchains/mbed_toolchain.py
@@ -639,7 +639,8 @@ class mbedToolchain(with_metaclass(ABCMeta, object)):
         dependencies = []
         buff = open(dep_path).readlines()
         if buff:
-            buff[0] = re.sub('^(.*?)\: ', '', buff[0])
+            # patched below buff[0] = re.sub('^(.*?)\: ', '', buff[0])
+            buff[0] = re.sub('^(.*?): ', '', buff[0])
             for line in buff:
                 filename = line.replace('\\\n', '').strip()
                 if filename:
